experiments/AMIP/moist_mpi_earth/coupler_utils/calendar_timer.jl

```diff
@@ -0,0 +1,52 @@
+# calendar timer
+#  - facilitates calendar functions and temporal interpolations
+
+"""
+    current_date(t)
+Return the model date
+"""
+current_date(t) = date0 + Dates.Second(t)
```

Comment:
Going forward, it will be best if `date0` is not a global variable. The question then is, what is the natural place to store this? There is some code in src regarding a clock that could be expanded or changed to support this!

Reply:
Yeah, there are a few places in the coming PRs where global variables are used (e.g. `landmask`). I can make an issue, so we don't forget to come back to this.